src/modules/Dashboard/features/Applets/AppletsTable/AppletsTable.hooks.test.ts

```diff
@@ -0,0 +1,153 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import { useContext } from 'react';
+import { act, renderHook } from '@testing-library/react';
+
+import { useAsync } from 'shared/hooks/useAsync';
+
+import { useAppletsDnd } from './AppletsTable.hooks';
+
+const mockAppletContextValue = {
+  rows: [],
+  fetchData: jest.fn(),
+};
+
+jest.mock('react', () => ({
+  ...jest.requireActual('react'),
+  useContext: jest.fn(),
+}));
+
+jest.mock('shared/hooks/useAsync', () => ({
+  useAsync: jest.fn(),
+}));
+
+jest.mock('api', () => ({
+  setFolderApi: jest.fn(),
+}));
+
+describe('useAppletsDnd', () => {
+  beforeEach(() => {
+    (useAsync as jest.Mock).mockReturnValue({ execute: jest.fn() });
+  });
+
+  afterAll(() => {
+    jest.clearAllMocks();
+  });
+
+  test('returns initial state', () => {
+    (useContext as jest.Mock).mockReturnValue(mockAppletContextValue);
+
+    const { result } = renderHook(() => useAppletsDnd());
```

Comment:
```suggestion
    const { result } = renderHook(useAppletsDnd);
```
please check if it works this way